submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoQueryService.kt

```diff
@@ -21,8 +22,8 @@ internal class SubmissionMongoQueryService(
     override fun existByAccNo(accNo: String): Boolean = submissionRepo.existsByAccNo(accNo)
```

Comment:
Please take a look to `SubmissionMongoQueryServiceTest``. I think it would be good to add a unit test there to check the expected exception is thrown when the submission doesn't exist